docs/guide/controller-start-up.md

```diff
@@ -2,10 +2,10 @@
 
 The .wbt file contains the name of the controller that needs to be started for each robot.
 The controller name is a platform and language independent field; for example when a controller name is specified as "xyz\_controller" in the .wbt file, this does not say anything about the controller's programming language or platform.
-This is done deliberately to ensure the independence between platform and programming language of .wbt files.
+This is done deliberately to ensure the platform and programming language independence of .wbt files.
 
-Now, when Webots tries to start a controller it must determine what programming language is used by that controller initially.
-So, Webots looks in the project's "controllers" directory for a subdirectory that matches the controller's name.
+So when Webots tries to start a controller it must first determine what programming language is used by this controller.
```

Comment:
```suggestion
When Webots tries to start a controller it must first determine what programming language is used by this controller.
```

Reply:
Thanks for your guidance and help, sir !!